### ISSUE TITLE
Automated testing isn't possible with auto sign in as it hangs on the create profile screen

### ISSUE THREAD

**Issue by oddgames-david** [playgameservices/play-games-plugin-for-unity#3297]
We need a way to disable auto login for firebase testlab and other automated testing services.

![image](https://github.com/playgameservices/play-games-plugin-for-unity/assets/1932174/ee107c30-7e3c-4262-95a1-e4730e61ce1d)

**Comment by oddgames-david** [comment]
Is play services being deprecated or something? No movement for a long while.

**Comment by smile616** [comment]
@oddgames-david what do you use for the tests? is it UI Automator 2.0 or Robotium?

**Comment by oddgames-david** [comment]
> @oddgames-david what do you use for the tests? is it UI Automator 2.0 or Robotium?

Game loop

**Comment by smile616** [comment]
thanks, I'll pass on the feature request.

**Comment by oddgames-david** [comment]
> thanks, I'll pass on the feature request.

yeah this plugin is old anyways. solution is to use version 10 for anybody reading this.

**Comment by NickAb** [comment]
If you are launching Game Loop tests via command line, try using `--no-auto-google-login`, see https://cloud.google.com/sdk/gcloud/reference/firebase/test/android/run#--auto-google-login.

If there is no logged in account, there should be no profile creation popup.

**Comment by oddgames-david** [comment]
> If you are launching Game Loop tests via command line, try using `--no-auto-google-login`, see https://cloud.google.com/sdk/gcloud/reference/firebase/test/android/run#--auto-google-login.
> 
> If there is no logged in account, there should be no profile creation popup.

thanks, will try this

**Comment by msclower** [comment]
Still a problem. Any sign of fix or solution. Thanks in advance.
